components/validator/guardian.go

```diff
@@ -213,7 +204,7 @@ func (g *Guardian) ConfirmTransaction(ctx context.Context, transactionId *big.In
 	return g.securityCouncilContract.ConfirmTransaction(txOpts, transactionId)
 }
 
-func (g *Guardian) outputRootAtBlock(ctx context.Context, l2BlockNumber uint64) (eth.Bytes32, error) {
+func (g *Guardian) OutputRootAtBlock(ctx context.Context, l2BlockNumber uint64) (eth.Bytes32, error) {
```

Comment:
It looks like u've removed all context arguments from the Guardian receiver, can we do something similar here?